monkey/monkey_island/cc/resources/auth/register.py

```diff
@@ -27,10 +28,19 @@ def post(self):
         Registers a new user using flask security register
 
         """
-        response: ResponseValue = register()
+        try:
+            username, password = get_username_password_from_request(request)
```

Comment:
Is there any reason why this can't be moved down where `username` and `password` are actually used? Same thing applies to the `Login` resource

Reply:
If the request is malformed then this should return `HTTP.BAD_REQUEST` and not `HTTP.INTERNAL_SERVER_ERROR` which will happen if we moved it down there.